LanguageFeatures/Extension-types/upper_bound_A01_t05.dart

```diff
@@ -22,8 +22,10 @@ import '../../Utils/static_type_helper.dart';
 extension type ET1<T>(T id) {}
 extension type ET2<T>(T id) {}
 
+typedef NullableNum = num?;
+
 extension type ET3<T extends String>(T id) implements ET1<T>, String {}
-extension type ET4<T extends int>(T id) implements ET2<T>, num? {}
+extension type ET4<T extends int>(T id) implements ET2<T>, NullableNum {}
```

Comment:
This should still be a compile-time error, so I'm afraid there won't be any LUB tests starting from extension types where we encounter a type of the form `T?` in the superinterface graph. But, of course, it's not a problem that we can't have a test for a case that doesn't exist.

(I suppose we already have some other test showing that it is an error for an extension type declaration to have an `implements ... T ...` where `T` is a type of the form `S?`, with or without aliasing.)

Reply:
No we don't have such tests. I've already made a note to myself to add them

Reply:
Fixed